testutils/sqlutils/conn.go

```diff
@@ -47,9 +48,9 @@ func PGUrl(t util.Tester, ts *server.TestServer, user, tempDir, prefix string) (
 
 	// Copy these assets to disk from embedded strings, so this test can
 	// run from a standalone binary.
-	tempCAPath, tempCACleanup := securitytest.RestrictedCopy(t, caPath, tempDir, "TestLogic_ca")
-	tempCertPath, tempCertCleanup := securitytest.RestrictedCopy(t, certPath, tempDir, "TestLogic_cert")
-	tempKeyPath, tempKeyCleanup := securitytest.RestrictedCopy(t, keyPath, tempDir, "TestLogic_key")
+	tempCAPath, tempCACleanup := securitytest.RestrictedCopy(t, caPath, tempDir, strings.Join([]string{user, prefix, "ca"}, "_"))
```

Comment:
You're exactly right. Fixed this properly by having PGUrl create a temporary directory every time.